app/src/main/java/com/bucketlist/ui/main/MainViewModel.kt

```diff
@@ -9,8 +9,8 @@ import androidx.lifecycle.viewModelScope
 import com.bucketlist.dto.BucketListItem
 
 class MainViewModel : ViewModel() {
+    //Use of Array list to get and set bucket list items
```

Comment:
Your code needs more documentation, so I added this to tell the reviewer or person updating your code later on what the code is doing because documentation is important.